### DISCUSSION TITLE
Not able to find the Correct Disconnect Reasons as like RDP Client disconnect reasons

### DISCUSSION THREAD

**Discussion by SolairajMariappan** [FreeRDP/FreeRDP#6740] [Q&A]
Not able to find the Correct Disconnect Reasons as like RDP Client disconnect reasons. like log off the machine, etc. 
https://social.technet.microsoft.com/wiki/contents/articles/37870.remote-desktop-client-troubleshooting-disconnect-codes-and-reasons.aspx

What is the correct way to find the right disconnect reasons ?

**Comment by akallabeth** [comment]
Currently not fully mapped, so you need to check source.
If someone is already doing that, adding the results to https://github.com/FreeRDP/FreeRDP/wiki/FAQ would be greatly appreciated.
